### DISCUSSION TITLE
Add dropdown to choose a KV that will be used in markdowns

### DISCUSSION THREAD

**Discussion by chook** [squidfunk/mkdocs-material#7022] [Q&A]
👋  For API Docs, we have multiple domains, one per env. 
I'm looking to create an experience where a user can select their region with a dropdown, and it will be registered in the URL as a /?region=A, and perhaps also in a cookie.

From there I'd like a javascript code (or similar) to change all markdowns that need domain to something like

The endpoint is: https://{{parse_endpoint(_region)}}:443/path/to/api

Would appreciate any help. New to mkdocs + mkdocs-material

**Comment by squidfunk** [comment]
Definitely possible with some extra JavaScript ☺️ Maybe somebody from the community has had a similar problem and can help out.
